File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Ingredient.java
package com.gasparaiciukas.owntrainer.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Ingredient {

    @SerializedName("quantity")
    @Expose
    private int quantity;
    @SerializedName("measureURI")
    @Expose
    private String measureURI;
    @SerializedName("foodId")
    @Expose
    private String foodId;

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public String getMeasureURI() {
        return measureURI;
    }

    public void setMeasureURI(String measureURI) {
        this.measureURI = measureURI;
    }

    public String getFoodId() {
        return foodId;
    }

    public void setFoodId(String foodId) {
        this.foodId = foodId;
    }

}

File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Ingredient.kt
package com.gasparaiciukas.owntrainer.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Ingredient
(
@SerializedName("quantity") @Expose var quantity: Int = 0,
@SerializedName("measureURI") @Expose var measureURI: String? = null,
@SerializedName("foodId") @Expose var foodId: String? = null
)